configure.ac

```diff
@@ -1926,7 +1926,7 @@ dnl -----
 dnl LTTng
 dnl -----
 if test "$enable_lttng" = "yes"; then
-  PKG_CHECK_MODULES([UST], [lttng-ust >= 2.10.0], [
```

Comment:
We just lowered it from 2.12 to 2.10. What platform are you using that only has 2.9?

Reply:
Debian 9.

Reply:
Alright